tests/sles4sap/saptune/mr_test.pm

```diff
@@ -103,6 +105,11 @@ sub post_fail_hook {
     if (get_var('PUBLIC_CLOUD_SLES4SAP')) {
         select_host_console(force => 1);
         my $args = OpenQA::Test::RunArgs->new();
+        record_info('Is the peering present?', join(' ', 'network_peering_present:', $self->{network_peering_present} // 'undefined'));
```

Comment:
Other `record_info` printing the same variable are using `'CONTEXT LOG'` in place of `'Is the peering present?'`